doc/ale.txt

```diff
@@ -1061,6 +1062,24 @@ g:ale_yaml_yamllint_options                       *g:ale_yaml_yamllint_options*
 
   This variable can be set to pass additional options to yamllint.
 
+------------------------------------------------------------------------------
+4.32. cmakelint                                  *ale-linter-options-cmakelint*
```

Comment:
I wasn't sure where to put this in the list as it doesn't seem to be alphabetical and changing the order would mean changing all of the section numbers. Is this okay?